app/views/course/condition/achievements/_form.html.slim

```diff
@@ -1,7 +1,11 @@
+- if @conditional.is_a?(Course::Achievement)
```

Comment:
I don't like this because this would eventually be a super long list of if-else.